src/Ring2-Compatibility.package/RG2MethodDefinition.extension/class/className.selector.isMetaSide..st

```diff
@@ -0,0 +1,12 @@
+*Ring2-Compatibility
+className: aString selector: aString2 isMetaSide: isMeta
```

Comment:
This change is strange... I don't understand why it is here, I did not change this method

Reply:
I was modifying the branch with with code for old Ring compatibility